tests/comm/test_mnnvl_moe_alltoall.py

```
@@ -0,0 +1,833 @@
+        world_size=world_size,
+    )
+
+    local_num_tokens = all_num_tokens[rank]
+    max_num_tokens = max(all_num_tokens)
+
+    # Generate inputs
+    token_selected_experts = generate_token_selected_experts(
+        local_num_tokens, ep_size, num_experts_per_rank, top_k
+    )
+
+    payloads, expert_id_payload_index = make_bfloat16_payloads(
```

Ruff (0.14.8)

Unpacked variable `expert_id_payload_index` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)